assets/js/base/components/product-list/index.js

```diff
@@ -10,20 +10,58 @@ import classnames from 'classnames';
 import Pagination from '@woocommerce/base-components/pagination';
 import ProductSortSelect from '@woocommerce/base-components/product-sort-select';
 import ProductListItem from '@woocommerce/base-components/product-list-item';
-import withProducts from '@woocommerce/base-hocs/with-products';
+import {
+	useStoreProducts,
+	useSynchronizedQueryState,
+} from '@woocommerce/base-hooks';
 import withScrollToTop from '@woocommerce/base-hocs/with-scroll-to-top';
 import './style.scss';
 
+const generateQuery = ( { sortValue, currentPage, attributes } ) => {
```

Comment:
Sorry for maybe silly question but does the order of params here make any difference for the caller?

Reply:
No, because it's an destructured object.  The caller can send in the object with the properties in any order.